ui/src/main/resources/FileManagerCode/Translations.fr.xml

```diff
@@ -53,6 +53,7 @@ fileManager.action.delete=Supprimer
 fileManager.action.deleteFilesConfirmation=Etes-vous certain de vouloir supprimer les fichiers sélectionnés ?
 fileManager.action.deleteFilesHint=Supprimer les fichiers sélectionnés
 fileManager.action.deleteFoldersConfirmation=Etes-vous certain de vouloir supprimer les dossiers sélectionnés ?
+fileManager.action.deleteError=Une erreur est survenue lors de la suppression (theError).
```

Comment:
That means that the localization is invoked at JS-time.
I wanted to avoid this and thus use Velocity-time message rendering.
I'll change `theError` to `__error__` however.

Reply:
I believe that solves the comment.
paul

Reply:
>> That means that the localization is invoked at JS-time.
> No. The localization is invoked from Velocity with the __error__ parameter which you then replace from JavaScript with the actual error message. But the translation message should have a parameter.

I meant here that using `{0}` would mean that we do localization at JS time whereas using __error__ allows the replacement to be done at JS time but localization to be done at velocity time. This works. 
Note that this means that we may have different languages used as the server's error text may be in a different language than that of the user. I think that this is ok.

Reply:
this has been discussed further and solved differently.